package.json

```
@@ -10,7 +10,11 @@
     "test:ui": "vitest --ui",
     "test:coverage": "vitest run --coverage",
     "lint": "eslint --ignore-path .gitignore --fix src",
-    "prettier": "prettier src --write"
+    "prettier": "prettier src --write",
+    "prepare": "cd aws-deployment && npm i",
+    "bootstrap": "npm run prepare && npm run build && cd aws-deployment && cdk bootstrap",
+    "deploy": "npm run bootstrap && cd aws-deployment && cdk deploy "
```

```suggestion
    "deploy": "cd aws-deployment && cdk deploy "
```

`cdk bootstrap` should only be run once. While `npm run deploy` command is used every time when re-deploy is needed. so it's not correct to include `npm run bootstrap` call in the `deploy` command.